src/Elmish.WPF/WpfProgram.fs

```diff
@@ -155,6 +158,17 @@ module WpfProgram =
     { program with LoggerFactory = loggerFactory }
 
 
+  /// Uses the specified handler for dispatch loop exceptions, dispatching the returned
+  /// messages when an exception occurs. The first (string) argument of onError is message
+  /// from Elmish describing the context of the exception, and may contain a rendered
+  /// message case.
+  ///
```

Comment:
The `ILogger` logs many different kinds of errors.  This handler you are adding only logs some of those errors.  I would like this name to indicate which subset of errors that it handles.

Reply:
I'm not sure exactly what you're after, but I updated the doc comment (before reading your comment here). Does that resolve this?

Reply:
The name that comes to my mind is `withElmishErrorHandler`.  How does that sound to you?

Reply:
Sure. I also updated the doc comment. How do you like it?